java-checks/pom.xml

```diff
@@ -134,12 +134,24 @@
                   <version>4.3.7.RELEASE</version>
                   <type>jar</type>
                 </artifactItem>
+                <artifactItem>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-email</artifactId>
+                    <version>1.3.2</version>
+                    <type>jar</type>
+                </artifactItem>
                 <artifactItem>
                   <groupId>org.springframework</groupId>
                   <artifactId>spring-beans</artifactId>
                   <version>4.3.7.RELEASE</version>
                   <type>jar</type>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>javax.mail</groupId>
+                  <artifactId>mail</artifactId>
+                  <version>1.4.7</version>
+                  <type>jar</type>
+                </artifactItem>
```

Comment:
Why do we need this dependency?

Reply:
Now, that I don't need to highlight on `Session` instance, I got rid of it.